pkg/apps/utils.jsx

```diff
@@ -91,7 +91,7 @@ export const show_error = ex => {
     if (ex.code == "cancelled")
         return;
 
-    if (ex.code == "not-found")
+    if (ex.code == "not-found" || ex.problem == "not-found")
```

Comment:
Can you explain this more?  `ex.code` is from PackageKit, and `ex.problem` is from cockpit-bridge.  The first is "not-found" when the package was not known to PackageKit, the latter is "not-found" when PackageKit itself isn't there.

Let's move this out into its own PR.

Reply:
To be honest, I did not dig into details of this. Test was failing here and while debugging I watched for content of the `ex`. I will need to start over to give proper answer to your question.

Reply:
Let's leave this out here, then.